en/development/debugging.rst

```diff
@@ -51,8 +51,13 @@ Using the Debugger Class
 
 .. php:class:: Debugger
 
-To use the debugger, first ensure that ``Configure::read('debug')`` is
-set to ``true``.
+To use the debugger, in your app/Config.php file, ensure that ``'debug' => filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),`` is set to ``true``.
+
+
+    Production Mode: set false. No error messages, errors, or warnings shown. Flash messages redirect.
```

Comment:
Flash message part is incorrect. It can be side effect but in most cases this is not the case.

Reply:
> Especially the debug mode part. So best never to mention this.

Why @dereuromark ?

Reply:
Because it is both incorrect and not needed info.